docs/usage/configuration-options.md

```diff
@@ -2065,7 +2065,7 @@ An additional use case is for GitLab users of project or group access tokens who
 If `ignorePrAuthor` is configured to true, it means Renovate will fetch the entire list of repository PRs instead of optimizing to fetch only those PRs which it created itself.
 You should only want to enable this if you are changing the bot account (e.g. from `@old-bot` to `@new-bot`) and want `@new-bot` to find and update any existing PRs created by `@old-bot`.
 
-Setting this field to `true` in GitLab will also mean that all Issues will be fetched instead of only those by the bot itself.
+Setting this field to `true` in Github, GitLab will also mean that all Issues will be fetched instead of only those by the bot itself.
```

Comment:
```suggestion
Setting this field to `true` in Github or GitLab will also mean that all Issues will be fetched instead of only those by the bot itself.
```